packages/client/src/objectSet/ObjectSetListenerWebsocket.ts

```diff
@@ -26,27 +26,22 @@ import {
   loadAllOntologies,
 } from "@osdk/client.unstable";
 import type {
-  FoundryObject,
-  Message,
-  Message_objectSetChanged,
-  ObjectSetSubscribeRequest,
-  ObjectSetSubscribeRequests,
+  ObjectSet,
+  ObjectSetStreamSubscribeRequest,
+  ObjectSetStreamSubscribeRequests,
   ObjectSetSubscribeResponses,
-  ObjectUpdate_object,
-  RefreshObjectSet,
+  ObjectSetUpdates,
+  OntologyObjectV2,
   StreamMessage,
-  StreamMessage_objectSetChanged,
   SubscriptionClosed,
-} from "@osdk/client.unstable.osw";
-import { batchEnableWatcher } from "@osdk/client.unstable.osw";
-import {
-  type ObjectSet,
-  type OntologyObjectV2,
 } from "@osdk/internal.foundry.core";
+
+import type { RefreshObjectSet } from "@osdk/client.unstable.osw";
 import * as OntologiesV2 from "@osdk/internal.foundry.ontologiesv2";
 import type { ConjureContext } from "conjure-lite";
 import WebSocket from "isomorphic-ws";
 import invariant from "tiny-invariant";
+import { ObjectSetSubscribeResponse_error } from "../../../client.unstable.osw/build/esm/generated/object-set-watcher/objectsetwatcher/api/ObjectSetSubscribeResponse.js";
```

Comment:
This should be a an import from `@osdk/client.unstable.osw` or somewhere else

Reply:
Yeah we don't need any of .unstavle.osw anymore, just a leftover dep that I mistakenly left in. Thank you for the catch!